app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="tab_text">25sp</dimen>
+    <dimen name="dp_50">50dp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.dp_50` appears to be unused
 Unused resources make applications larger and slow down builds.